docs/NOW-spec.md

```diff
@@ -1013,3 +1026,6 @@ packet-beta
 ### Version History
 - 1.0
     - Initial protocol version
+- 1.1
+    - Add IO redirection capability flag and explicit IO redirection flags for exec messages.
+    - Add working directory specification for Run (ShellExecute) messages.
```

Comment:
issue: You should also document the modifications to the on-wire encoding.

Reply:
It its about https://github.com/Devolutions/now-proto/pull/29#discussion_r2170888484, then on-wire encoding was not changed, only encoding change is new field for `Run` command and new flags which are described

Reply:
Okay, I saw your comment! My only remark is that it would be easier to not mix this kind of fixes with other unrelated changes